common/src/main/java/com/tc/net/protocol/transport/ClientConnectionEstablisher.java

```diff
@@ -285,32 +291,28 @@ private boolean tryToConnect(boolean connected, Supplier<Boolean> stopCheck) {
     return !connected && !stopper && !stopped;
   }
 
-  void handleConnectException(Exception e, boolean logFullException, Logger logger) {
+  void handleConnectException(Exception e, boolean logFullException, Logger logger) throws InterruptedException {
     if (logger.isDebugEnabled() || logFullException) {
       logger.error("Connect Exception", e);
     }
 
     if (CONNECT_RETRY_INTERVAL > 0) {
```

Comment:
Logic in the constructor means this should never be < 1000. Think we can safely discard this if check.

Reply:
removed in the latest commit